### ISSUE TITLE
[Image Optimizer] Argument 1 passed to ThemeHouse\ImageOptimizer\ContentHandler\ImageProxy::hasImage() must be an instance

### ISSUE THREAD

**Issue by Nirjonadda** [Audentio/xf2addon-issues#1982]
Server error log

```
TypeError: Argument 1 passed to ThemeHouse\ImageOptimizer\ContentHandler\ImageProxy::hasImage() must be an instance of XF\Mvc\Entity\Entity, null given, called in /home/nadda/public_html/src/addons/ThemeHouse/ImageOptimizer/ContentHandler/AbstractHandler.php on line 43 src/addons/ThemeHouse/ImageOptimizer/ContentHandler/ImageProxy.php:90 

Generated by: Unknown account Sep 8, 2019 at 3:49 AM 

Stack trace

#0 src/addons/ThemeHouse/ImageOptimizer/ContentHandler/AbstractHandler.php(43): ThemeHouse\ImageOptimizer\ContentHandler\ImageProxy->hasImage(NULL)
#1 src/addons/ThemeHouse/ImageOptimizer/Cron/Optimize.php(18): ThemeHouse\ImageOptimizer\ContentHandler\AbstractHandler->optimize(Object(ThemeHouse\ImageOptimizer\Entity\Status))
#2 [internal function]: ThemeHouse\ImageOptimizer\Cron\Optimize::optimizeImages(Object(XF\Entity\CronEntry))
#3 src/XF/Job/Cron.php(34): call_user_func(Array, Object(XF\Entity\CronEntry))
#4 src/XF/Job/Manager.php(253): XF\Job\Cron->run(G)
#5 src/XF/Job/Manager.php(195): XF\Job\Manager->runJobInternal(Array, G)
#6 src/XF/Job/Manager.php(79): XF\Job\Manager->runJobEntry(Array, G)
#7 job.php(42): XF\Job\Manager->runQueue(false, 8)
#8 {main}

Request state

array(4) {
  ["url"] => string(8) "/job.php"
  ["referrer"] => string(32) "/admin.php"
  ["_GET"] => array(0) {
  }
  ["_POST"] => array(0) {
  }
}
```

**Comment by JakeBooher** [comment]
What version of the Image Optimizer add-on are you currently running?

**Comment by Nirjonadda** [comment]
[TH] Image Optimizer 1.1.1 is running but now updated to 1.1.1 Patch Level 1, I will let you know if we have any more this Server error log.

**Comment by JakeBooher** [comment]
This was fixed in 1.1.1 Patch Level 1, so it should no longer be an issue
